src/services/demo.ts

```diff
@@ -0,0 +1,18 @@
+import { Store } from 'babydux'
```

Comment:
Can you give a bit more context on what this "demo" part is doing?

Reply:
From the commit message:

> start app with demo data loaded for faster testing

The app now starts up with some demo providers, service areas, and rep points loaded. When development slows down, we can either remove it or update it with more realistic demo data to show customers.